### DISCUSSION TITLE
Installing npm modules taking too long

### DISCUSSION THREAD

**Discussion by bcostaaa01** [jaredpalmer/tsdx#1163] [Q&A]
Hey @jaredpalmer 👋

I have been trying to create a new package using TSDX, but it does not seem to go through the installation of the npm modules... It is taking way too long and therefore I would like to ask if this is normal or if this is an issue on my end or TSDX itself?

<img width="373" alt="Screenshot 2022-12-22 at 13 22 22" src="https://user-images.githubusercontent.com/72168158/209144066-2975b15a-8c16-4c60-9e2d-e27dcfecfd6b.png">

I would appreciate an answer from you regarding this issue.

The concept looks really interesting by the way, hence why I would love to try it out!👏

Bruno

**Comment by leamandeep** [comment]
I'm too facing this issue
